### PR TITLE
added i18n support

### PR DESCRIPTION
**Pull request by imscaradh**
@Tschanzt

### CODE REVIEW

File: app/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+msgid "Daily"
+msgstr "Täglich"
+
+#: templates/app/index.html:24
+msgid "Last two weeks"
+msgstr "Letzte zwei Tage"
+
+#: templates/app/login.html:14
+msgid "Your username and password didn't match. Please try again."
+msgstr ""
```
Comment (tschanzt) [inline app/locale/de/LC_MESSAGES/django.po:60]: @imscaradh The Translation is missing

File: app/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,103 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```
Comment (tschanzt) [inline app/locale/de/LC_MESSAGES/django.po]: @imscaradh The po file is fuzzy please resolve this issue.

**Comment by tschanzt** [comment]
@imscaradh otherwise the pr looks fine.

**Comment by imscaradh** [comment]
@Tschanzt should be ok

**Pull request merged**